routing/localchans/manager.go

```diff
@@ -106,6 +111,36 @@ func (r *Manager) UpdatePolicy(newSchema routing.ChannelPolicy,
 		return err
 	}
 
+	// Return an error if any of the channels in chansToUpdate
+	// were not found.
+	if len(chansToUpdate) > 0 {
+		var errStr string
+		for _, chanPoint := range chanPoints {
+			channel, err := r.FetchChannel(chanPoint)
+			if errors.Is(err, channeldb.ErrChannelNotFound) {
+				errStr += fmt.Sprintf(
+					"channel %s:%d not found; ",
+					chanPoint.Hash.String(), chanPoint.Index,
+				)
+				continue
+			}
+			if err != nil {
+				errStr += fmt.Sprintf(
+					"error fetching channel %s:%d; ",
+					chanPoint.Hash.String(), chanPoint.Index,
+				)
+				continue
+			}
+			if channel.IsPending {
+				errStr += fmt.Sprintf("channel %s:%d is "+
+					"not yet confirmed; ",
+					chanPoint.Hash.String(), chanPoint.Index)
+				continue
+			}
```

Comment:
What about the rest of the channels that were not updated but dont fall into the category above? Maybe add a generic error message here for those